web_sheet_full_width/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Show sheets with full width",
+    "version": "11.0.1.0.1",
+    "author": "Therp BV, Sudokeys, GRAP, Métal Sartigan, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "summary": "Use the whole available screen width when displaying sheets",
+    "category": "Tools",
+    "depends": [
+        'web',
+    ],
+    "data": [
+        "view/qweb.xml",
```

Comment:
I think this should be `templates/assets.xml` instead.

Reply:
I thought it was weird... I'll sure rename this.